packages/astro/src/default/utils/content.ts

```diff
@@ -30,6 +30,7 @@ export async function getTutorial(): Promise<Tutorial> {
       tutorialMetaData.template ??= 'default';
       tutorialMetaData.i18n = Object.assign({ ...DEFAULT_LOCALIZATION }, tutorialMetaData.i18n);
       tutorialMetaData.openInStackBlitz ??= true;
+      tutorialMetaData.downloadAsZip ??= true;
```

Comment:
Oh I had an idea, what if instead of defaulting to `true` here which would add it to existing project that do an upgrade, we add the `downloadAsZip: true` attribute to the tutorial metadata for new project. So here:

https://github.com/stackblitz/tutorialkit/blob/5a2a64f35be823c7451af898e4bb6f91693fabff/packages/cli/overwrites/src/content/tutorial/meta.md?plain=1#L1-L6

What do you think?

Reply:
Yup sounds good! Let's default to `false` and configure new projects to use it.